### PR TITLE
[B01]  Create input component

### PR DESCRIPTION
**Pull request by KarolinaKopacz**
## What does the PR do?

Create component with text input with send button for chat.

### CODE REVIEW

File: src/components/NewMessageInput.tsx [hunk 1]
```diff
@@ -0,0 +1,52 @@
+import React, { useState } from "react";
+import { Button, FormControl, InputGroup, Spinner } from "react-bootstrap";
+
+interface Props {
+  isLoading: boolean;
+  buttonText: string;
+}
+
+const NewMessageInput = (props: Props) => {
```
Comment (Fortidude) [inline src/components/NewMessageInput.tsx]: `ChatTextInput` or `ChatInputFooter`

File: src/components/NewMessageInput.tsx [hunk 2]
```diff
@@ -0,0 +1,52 @@
+import React, { useState } from "react";
+import { Button, FormControl, InputGroup, Spinner } from "react-bootstrap";
+
+interface Props {
+  isLoading: boolean;
```
Comment (Fortidude) [inline src/components/NewMessageInput.tsx]: `isLoading` should be managed by the send message function. Currently looks like `handleSendMessage` is inside this component, but `isLoading` outside.

**Review by Fortidude — CHANGES_REQUESTED** [review]

File: src/components/NewMessageInput.tsx (hunk 1 above)
Comment (KarolinaKopacz) [reply src/components/NewMessageInput.tsx]: Done!

File: src/components/NewMessageInput.tsx (hunk 2 above)
Comment (KarolinaKopacz) [reply src/components/NewMessageInput.tsx]: Done!

**Pull request merged**